src/index.html

```diff
@@ -25,7 +25,7 @@
 		  
 		  <section id="todoList"></section>
 		  
-		  <div class="clearAll">Clear all completed</div>
+		  <button class="clearAll">Clear all completed</button>
```

Comment:
- [ ] You have implemented `Clear All Completed` function in `index.js`. This function is for deleting all selected lists. Now as per the requirement you need to implement another function and button to delete a specific list.

- ***Requirement***
![image](https://user-images.githubusercontent.com/60083437/221003436-2195b0e8-be01-444c-bfe2-5832b5f5c11f.png)

- ***Suggestion on your design***
![image](https://user-images.githubusercontent.com/60083437/221003885-0147cb03-5352-4efe-8274-c1c623d8e120.png)